ts/nni_manager/rest_server/index.ts

```diff
@@ -64,7 +63,8 @@ export class RestServer {
         const app = express();
         expressWs(app, undefined, { wsOptions: { maxPayload: 4 * 1024 * 1024 * 1024 }});
 
-        app.use('/' + this.urlPrefix, rootRouter());
+        app.use('/' + this.urlPrefix, mainRouter());
+        app.use('/' + this.urlPrefix, fallbackRouter());
```

Comment:
maybe a stupid question, one path or url can be registered with two handlers?

Reply:
[Yes](https://expressjs.com/en/guide/routing.html)